src/frame/js/smooch.jsx

```diff
@@ -216,6 +216,14 @@ export function login(userId = '', jwt, attributes) {
             }
         }
     })).then((loginResponse) => {
+        Raven.setUserContext({
+            id: loginResponse.appUser.userId || loginResponse.appUser._id
```

Comment:
I was hesitant to use `userId` in case it was an email.... should we just use `_id`?